### PR TITLE
Add Pull request template

### PR DESCRIPTION
**Pull request by Vanshiii203**
issue #139

### CODE REVIEW

**Comment by Vanshiii203** [comment]
https://user-images.githubusercontent.com/94471189/197375691-6e32d5fb-8f41-447b-a7d0-87133d7398ec.mp4

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+<!----Please delete options that are not relevant. And in order to tick the check box just put x inside them for example [x] like this----->
+
+- [] Bug fix (non-breaking change which fixes an issue)
+- [] New feature (non-breaking change which adds functionality)
+
+## Checklist:
+
+<!----Please delete options that are not relevant.And in order to tick the check box just put x inside them for example [x] like this----->
+
+- [] I have made this from my own.
```
Comment (Kanika637) [inline .github/PULL_REQUEST_TEMPLATE.md:20]: I don't think we require this line, kindly remove it.

**Review by Kanika637 — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (Vanshiii203) [reply .github/PULL_REQUEST_TEMPLATE.md:20]: okay

**Comment by Vanshiii203** [comment]
@Kanika637 I have removed that line

**Review by Kanika637 — APPROVED** [review]
LGTM🚀
Great Work🚀
Don't forget to give a ⭐ to the repo, if you learned something.

**Pull request merged**